docs/guides/building/windows-installer.md

```diff
@@ -0,0 +1,222 @@
+# Windows Installer
+
+Tauri applications for Windows are distributed as Microsoft Installer (`.msi` files). The Tauri CLI bundles your application binary and additional resources in this format if you build on Windows. This guide provides information about available customization options for the installer.
```

Comment:
> Tauri applications for Windows are distributed as Microsoft Installer (`.msi` files).

this one sounds like _something_ is missing. Maybe change it to "Microsoft Installer**s**" or "Microsoft Installer (`.msi`) files", or something. Maybe it's just my german brain trippin here 😂

Reply:
```suggestion
Tauri applications for Windows are distributed as a Microsoft Installer (`.msi` files). The Tauri CLI bundles your application binary and additional resources in this format if you build on Windows. This guide provides information about available customization options for the installer.
```

Reply:
Yeah, no, don't like that one either

Reply:
I think just making it plural works. I also think adding more to the sentence like "that extract the required files to their installation destination" may be beneficial